unitTest/@testFEMComputationalFluidDynamicsAnalysis/testFEMComputationalFluidDynamicsAnalysis.m

```diff
@@ -15,5 +15,6 @@
     methods (Test)
         testFEM4NavierStokesSteadyState2D(testCase)
         testFEM4NavierStokesTransientBossak3D(testCase)
+        testFEM4NavierStokesSteadyState2DReferencePaper(testCase)
```

Comment:
Change the name of the function testFEM4NavierStokesSteadyState2DReferencePaper to testFEM4NavierStokesSteadyStateFlowAroundCylinder2D